database/exip.sql

```diff
@@ -154,7 +154,7 @@ CREATE TABLE `Application` (
 	`dealType` varchar(4) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT 'EXIP',
   `policyContact` varchar(191) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
   `nominatedLossPayee` varchar(191) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
-  `migratedV2toV3` tinyint(1) DEFAULT NULL,
```

Comment:
should we not keep this flag for auditing purposes?

Reply:
I don't think it serves any value in all aspects. If we need to determine historic data migration versions from an auditing perspective, we can determine this from particular dates in the applications and release versions.

Also, if we were to keep the previous migration flag it:
- Could be confusing/misleading because e.g `migratedV1toV2` would not exist.
- The table would have a lot of "dead columns".

If this was a very high volume service and/or applications are long standing, then I would agree it's worth keeping all migration version flags in a new table, but at the moment, it is very low volume and low/no risk.

Also all data migration is quite minor really, excluding `migratedV1toV2` which had 100x more changes than V2 to V3 or V3 to V4.

Reply:
@abhi-markan Can we get your input on this please.

Reply:
Is this migrating all applications?

Would it be possible to change the column to `migration` and specify the version i.e. `2`, `3` and etc.

Agree with @Zainzzkk we need to keep track of which application was migrated under which version.

Reply:
Discussed offline - in summary:

- I've updated the script so we have a `migratedTo` integer field.
  - In progress applications will have this set to 4.
  - If any other applications have a `migratedV2toV3` value of 1 (`true`), `migratedTo` will be set to 3.
- `migratedV2toV3` has been deprecated.

RE migration - the data structure is changing for all applications regardless. However, only new data will be added for applications that are in progress. They will essentially gain an additional `DeclarationModernSlavery` relationship with the application, as this are now required questions.